ocs_ci/helpers/helpers.py

```diff
@@ -716,6 +716,76 @@ def create_multiple_pvcs(
     return ocs_objs
 
 
+def create_bulk_pvcs(
```

Comment:
why did you duplicate the create_multiple_pvc() function (on line 639) and not modifying it ?
all you need is to change the return value.

Reply:
Agree. Added to the existing function returning directory path, and change calls in the relevant places in the project.